cartesian2polar.py

```diff
@@ -0,0 +1,14 @@
+from math import acos, sqrt, pi
+
+# Read values for x and y
+x = float(input("x = "))
+y = float(input("y = "))
+
+# Convert polar coordinates to cartesian
+r = sqrt(x**2 + y**2)
+θ = acos(x/r)
+θ *= 1 if y >= 0 else -1

```

Comment:
**Suggestion:** The conversion from cartesian to polar coordinates is not correct when x is negative. The acos function returns a value between 0 and pi, so when x is negative, the angle should be 2*pi - acos(x/r) instead of -acos(x/r).
```suggestion
r = sqrt(x**2 + y**2)
if x >= 0:
    θ = acos(x/r)
else:
    θ = 2*pi - acos(x/r)
```